src/timeMachine/components/TimeMachineFluxEditor.tsx

```diff
@@ -102,15 +102,18 @@ const TimeMachineFluxEditor: FC = () => {
       row = currentRange.endLineNumber + 1
     }
 
-    let text = ''
+    // FIXME: now that we are using FluxDocs, not hardcoded FLUX_FUNCTIONS
+    // we are missing the example in the http request payload
+    let text = !!(func as FluxToolbarFunction).example
+      ? (func as FluxToolbarFunction).example
+      : `${func.name}()`
+
```

Comment:
Another instance of this incorrect use of `fn.name`.
Must come up with a better solution.

Reply:
`${func.example}.${func.name}()`